### ISSUE TITLE
PoK upgrades?

### ISSUE THREAD

**Issue by PhilRoi** [daveah/ti4_planet_selection#13]
Any plans for udating this tool for PoK when it hits?

**Comment by daveah** [comment]
I do fully intend to update for the new expansion.  As soon as I see the planet details I'll be able to update - the way I designed the code it will be trivial to do.  I'll also be adding new configurations for 7/8 players and for 3/4 ring galaxies as appropriate.  At the moment I'm anticipating similar configurations to the expansions for TI3 - but am also expecting to be surprised.
Another thing I'll consider, depending on the exact details of what drops, is configurations that favour high or normal resource allocations.  The group I played TI3 with always played in special high resource configurations.  It slowed the game down a bit, but meant you got deeper into tech trees and had more epic battles overall.
I'm a regular TI4 player myself, and I always use this tool, so expect to see it actively developed.  Glad to see you are using it.
